onelogin_aws_cli/__init__.py

```diff
@@ -221,42 +196,3 @@ def save_credentials(self):
         # Reset state in the case of another transaction
         self.token = None
         self.credentials = None
-
-    @staticmethod
```

Comment:
The logic for `generate_config()` is now found in `onelogin_aws_cli.ConfigurationFile.initialise()`